### DISCUSSION TITLE
Is hard to focus and start writting in milkdown editor initially

### DISCUSSION THREAD

**Discussion by sytolk** [Milkdown/milkdown#1319] [Q&A] [answered]
To repeat this issue you can 
1. delete all content of milkdown editor here: https://milkdown.dev/playground
2. lose focus in milkdown editor (click on codemirror editor for example)
3. click in middle of milkdown editor and try to start writting content..

You can get focus of editor only if you click on the first line but its hard to find the exact place..

I think that better fix for this issue will be to add style height:'100%' of first 3 div's? 
like this: 
```html
<div data-milkdown-root="true" style="
    height: 100%;
">
<div class="milkdown" style="
    height: 100%;
">
<div contenteditable="true" translate="no" class="ProseMirror prose dark:prose-invert outline-none milkdown-theme-nord editor" role="textbox" aria-expanded="true" style="
    height: 100%;
"></div></div></div>
```
The question is how can I add this style with tilewindcss? and can we change website source code with this @Saul-Mirone ?

**Comment by Saul-Mirone** [comment, marked as answer]
Yeah, I'm totally okay with the change. PR welcome.

**Reply by sytolk** [reply]
https://github.com/Milkdown/website/pull/37
